benchmarks/maxtext_xpk_runner.py

```diff
@@ -53,7 +53,10 @@ class PathwaysConfig:
   server_image: str
   proxy_image: str
   runner_image: str
-  remote_python_sidecar_image: str
+  remote_python_sidecar_image: str = None
```

Comment:
Nit: Can we keep the initializations consistent?

Reply:
Yup, we tested without Remote Python on v6e. For better consistency I default the server/proxy/image to None. By default we do put values into server/proxy/runner image https://github.com/AI-Hypercomputer/maxtext/blob/8350feff91d8072c0fa9537bcf32b1d846c9a3b2/benchmarks/benchmark_runner.py#L251-L257

And

https://github.com/AI-Hypercomputer/maxtext/blob/8350feff91d8072c0fa9537bcf32b1d846c9a3b2/benchmarks/benchmark_runner.py#L111-L135